main/test/flix/Test.Exp.Regions.flix

```diff
@@ -80,7 +84,7 @@ namespace Test/Exp/Regions {
     @test
     def testRegions1Region01(): Bool =
         region r {
-            let a = [1; 8];
+            let a = $ARRAY_REPEAT$(r, 1, 8);
```

Comment:
Can we use `Array.new` in this file?

Reply:
I suppose we could. Just wanted to reduce dependency on the std lib but if we're already using mutmap and mutlist then we may as well just use array

Reply:
Great